Spec/Auth.swift

```diff
@@ -693,17 +692,17 @@ class Auth : QuickSpec {
 
                 it("overriding defaults") {
                     // Custom values
-                    let expectedClientId = "token_client"
                     let expectedTtl = 4800.0
                     let expectedCapability = "{\"canpublish:*\":[\"publish\"]}"
 
-                    let tokenParams = ARTTokenParams(clientId: expectedClientId)
+                    let tokenParams = ARTTokenParams(clientId: currentCliend)
```

Comment:
It's the client ID passed to the `rest` instance used below. RSA15c says that the client ID can't change in a TokenRequest or TokenDetails from the one passed in the ClientOptions, so we can't expect it to be different.